torch/csrc/jit/codegen/cuda/ops/normalization.cpp

```diff
@@ -587,8 +587,11 @@ ForwardNormResult batch_norm(
     auto invstd_bcast = broadcast(unbiased_invstd, broadcast_mask);
 
     // During inference, mean/invstd output are empty tensors
-    mean = TensorViewBuilder().shape(std::vector<int64_t>{0}).build();
-    invstd = TensorViewBuilder().shape(std::vector<int64_t>{0}).build();
+    // on CPU, but not on CUDA. We need to make sure we have the same
+    // behavior as with eager mode on CUDA.
+    mean = set(running_mean); // use set to avoid "trivial input forwarding NOT
```

Comment:
https://github.com/pytorch/pytorch/blob/fe87ae692f813934d1a74d000fd1e3b546c27ae2/aten/src/ATen/native/Normalization.cpp#L676-L677

Why is it returning saved_mean/var for inference mode?! this looks very wrong to me. I'm wondering if there's any issue with the tests?!?!

Reply:
If we are running in inference with running_stats provided, there's no good reason for us to save running_stats out.

Reply:
so... Just realized that our devel branch started to hit issues with the mismatch here. :facepalm: 
Let's keep your changes. But we should discuss the cuda behavior.... why do we need to save them....

Reply:
FYI, since we checked in trivial forwarding, we might want to remove `set` here.